### PR TITLE
[Trivial] Improve end-of-namespace comment consistency

### PR DESCRIPTION
**Pull request by meshcollider**
Continuing on from #9544 this just makes the end-of-namespace comments consistent across all files

While purely style changes are not normally acceptable, this one should cause very few merge conflicts just because namespaces aren't usually touched often, and because of the nature of namespace indentation, I believe its quite a readability improvement to keep track of what the closing braces belong to.

### CODE REVIEW

**Comment by laanwj** [comment]
Meh, why all the end-of-namespace pulls lately? We don't have a clear rule for these, no need to have a PR every three days that updates them.
Also `src/leveldb` changes should go into the appropriate upstream repository.

**Comment by MeshCollider** [comment]
Just saw #9544 hadn't finished the job, because that was merged then this just makes it all consistent.
Good point on the leveldb though

**Comment by MeshCollider** [comment]
Removed changes to src/leveldb/

**Comment by sipa** [comment]
The style guide says that braces must be on the next line for namespaces, classes, methods, and functions.

**Comment by MeshCollider** [comment]
Almost all the existing namespaces have the brace on the same line, would you prefer if they were all changed in this PR too?

**Comment by sipa** [comment]
I don't think there is a need for a massive change one way or another, but certainly not for a change in the wrong direction.

**Comment by MeshCollider** [comment]
Refactored as per style guide, no extra effort to get it fixed really :)

**Review by promag — CHANGES_REQUESTED** [review]
Missing documentation fix for https://github.com/bitcoin/bitcoin/commit/f3f1e2e7d38005fa3edecb01469a18a6f7500ea6.

**Comment by MeshCollider** [comment]
@promag What do you mean?

File: src/memusage.h [hunk 1]
```diff
@@ -17,8 +17,7 @@
 
 #include <boost/foreach.hpp>
 
-namespace memusage
-{
+namespace memusage {
```
Comment (promag) [inline src/memusage.h]: Remove change?

**Review by promag — CHANGES_REQUESTED** [review]
At the end I would keep two commits like:
 - Fix namespace brace style (includes documentation fix)
 - Improve end-of-namespace comment consistency

Other than these, ACK.

**Comment by MeshCollider** [comment]
Decided to squash them all into one commit because its all refactoring, and the previous commits were already squashed. Thanks for pointing out that missed brace @promag

**Comment by fanquake** [comment]
Going to close this for now. You could rebase, extend the changes and reopen just after the next major branch off, as that is usually the best time to merge commits that touch a lot of files.

**Pull request closed without merging**